src/cli.ts

```diff
@@ -1,7 +1,66 @@
-function main(packageRootDir: string) {
+import meow from 'meow';
+import * as fs from 'fs';
+import pify from 'pify'
+
+import {generatePackageTree} from './package-tree';
+import {outputToUser} from './output';
+
+const cli = meow({
+  help:`  
+    Usage
+      $ TODO <project>
+ 
+    Options
+      --help        Prints this help message.
+  `,
+  flags: {
+    help: {type: 'boolean'}
+  }
+});
+
+if(cli.input.length !== 1){
+  console.error(`Error: should have 1 argument, but got ${cli.input.length}`)
+  cli.showHelp(1);
+  process.exit(1);
+}else{
+  run(cli.input[0]);
+}
+
+async function validNodePackage(path: string){
+  try{
+    await pify(fs.stat)(path)
```

Comment:
Missing semicolon

Reply:
(They're optional in JavaScript but our coding style / standards enforce them)

Reply:
done